packages/core/src/external/fhir/resources/condition.ts

```diff
@@ -0,0 +1,71 @@
+import { CodeableConcept } from "@medplum/fhirtypes";
+import {
+  CONDITION_CLINICAL_STATUS_URL,
+  CONDITION_VERIFICATION_STATUS_URL,
+} from "@metriport/shared/medical";
+
+export type ConditionClinicalStatus =
```

Comment:
can we use `keyOf` for typing here instead of redefining a type in addition to the object?

Reply:
It's a bit more error prone to solely rely on the keys of an untyped object as the status codes, but I refactored this to make it closer to the other examples in the codebase with use `const a = ["code1", "code2"] as const; type A = (typeof a)[number]` which is also more standard practice in TS.